src/components/Dashboard/ProgressBar/ProgressBar.js

```diff
@@ -1,15 +1,48 @@
-import React from "react";
+import React, { Component } from "react";
 import styles from "./ProgressBar.scss";
 
-const ProgressBar = () => {
-  return (
-    <ul className={styles.ProgressBar}>
-      <li>Dane dziecka</li>
-      <li>Wnioskodawcy</li>
-      <li>Zespół</li>
-      <li>Podsumowanie</li>
-    </ul>
-  );
-};
+class ProgressBar extends Component {
+  render() {
+    let menuStep1;
+    let menuStep2;
+    let menuStep3;
+    let menuStep4;
+
+
+    switch (this.props.pageCounter) {
+      case 0:
+        menuStep1 = styles.active;
+        break;
+      case 1:
+        menuStep2 = styles.active;
+        menuStep1 = styles.active;
+        break;
+      case 2:
+        menuStep3 = styles.active;
+        menuStep2 = styles.active;
+        menuStep1 = styles.active;
+        break;
+      case 3:
+        menuStep4 = styles.active;
+        menuStep3 = styles.active;
+        menuStep2 = styles.active;
+        menuStep1 = styles.active;
+        break;
+    }
+
+
+    return (
+      <div className={styles.ProgressBar}>
+        <ul className={styles.ProgressBar__list}>
```

Comment:
we use css modules (https://github.com/CodeForPoznan/alinka-electron/blob/master/README.md) so instead of using some fusion of css modules with BEM, you could just name it as: `styles.List` and when you check React devtools, this class name will be then: `ProgressBar__List__[random-characters]`, right now it looks like this `ProgressBar__ProgressBar__list__[random-characters]`